src/ components/EateryCard.js

```diff
@@ -0,0 +1,244 @@
+  CardActions,
+  Avatar,
+  Dialog,
+} from "@mui/material";
+import ExpandMoreIcon from "@mui/icons-material/ExpandMore";
+
+const StyledCard = styled(Card)({
+  backgroundColor: "#23272A",
+  border: "2px solid rgba(0, 0, 0, 0.2)",
+  textAlign: "left",
+  borderRadius: 7,
+  height: "100%",
+  justifyContent: "flex-start"
+});
+
+const StyledCardHeader = styled(CardHeader)({
+  fontWeight: 500,
+  backgroundColor: "#1D1F21",
+});
+
+const NameText = styled(Typography)({
+  color: "white",
+  padding: 0,
+  fontFamily:
+    '"Zilla Slab", "Inter", -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen", "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue", sans-serif',
+  textTransform: "capitalize",
+});
+
+const LocationText = styled(Typography)({
+  color: "#6C757D",
+  marginBottom: "10px",
+  fontWeight: 500,
+  fontSize:14
+});
+
+const DescriptionText = styled(Typography)({
+  color: "white",
+});
+
+const OpenText = styled(Typography)({
+  color: "#19b875",
+  fontSize: 14,
+  fontWeight:500,
+  fontFamily: '"Inter", -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen", "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue", sans-serif'
+});
+
+const ClosedText = styled(Typography)({
+  color: "#dd3c18",
+  fontSize: 14,
+  fontWeight:500,
+  fontFamily: '"Inter", -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen", "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue", sans-serif'
+});
+
+const ActionButton = styled(Button)({
+  fontFamily:
+    '"Zilla Slab", "Inter", -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen", "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue", sans-serif',
+  color: "white",
+  backgroundColor: "#1D1F21",
+  elevation: 30,
+});
+
+const GreenDot = styled(Card)({
+  "@keyframes blinking": {
+    "0%": {
+      opacity: 0,
+    },
+
+    "50%": {
+      opacity: 1,
+    },
+
+    "75%": {
+      opacity: 1,
+    },
+
+    "100%": {
+      opacity: 0,
+    },
+  },
+  background: "#19b875",
+  width: "100%",
+  height: "100%",
+  foregroundColor: "#19b875",
+  animationName: "blinking",
+  animationDuration: "1s",
+  animationIterationCount: "infinite",
+});
+
+const RedDot = styled(Card)({
+  "@keyframes blinking": {
+    "0%": {
+      opacity: 0,
+    },
+
+    "50%": {
+      opacity: 1,
+    },
+
+    "75%": {
+      opacity: 1,
+    },
+
+    "100%": {
+      opacity: 0,
+    },
+  },
+  background: "#dd3c18",
+  width: "100%",
+  height: "100%",
+  foregroundColor: "#dd3c18",
+  animationName: "blinking",
+  animationDuration: "1s",
+  animationIterationCount: "infinite",
+});
+
+const SpecialsContent = styled(Accordion)({
+  backgroundColor: "#23272A",
+});
+
+export default function EateryCard({
+  name,
+  location,
+  description,
+  menuURL,
+  todaysSpecials,
+  acceptsOnlineOrders,
+  isOpen,
+  statusMsg,
+  todaysSoups,
+}) {
+  const [modalOpen, setModalOpen] = useState(false);
```

Comment:
Name boolean values with is or another such word (has, should, etc) i.e. `isModalOpen, setIsModalOpen`